src/components/menu/menu.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "./menu.module.scss";
+
+export const Menu = () => {
+  const menu = ["all", "documentary", "comedy", "horror", "crime"];
```

Comment:
I strongly recommend to place constants out of components for 2 reasons:
1. Your component does not need to re-create it each time per re-render.
2. You can reuse them later in any part of project.

In this case constants have specific reason - they belong to single component. So, you can easily put them out of component (above it)